src/dune/dune_env.ml

```diff
@@ -105,6 +121,10 @@ module Stanza = struct
           User_error.raise ~loc
             [ Pp.textf "Variable %s is specified several times" k ] )
 
+  let odoc_field =
+    field "odoc" ~default:Odoc.empty
+      (Dune_lang.Syntax.since Stanza.syntax (2, 0) >>> Odoc.decode)
```

Comment:
```suggestion
      (Dune_lang.Syntax.since Stanza.syntax (2, 2) >>> Odoc.decode)
```

Since this will be part of Dune 2.2. You also need to update the variable `latest_version` to `(2, 2)` in `stanza.ml`